app/models/observation.rb

```diff
@@ -1,4 +1,8 @@
 class Observation < ActiveRecord::Base
   belongs_to :observer, class_name: "Dog"
   belongs_to :observed, class_name: "Dog"
+
+  scope :matches, ->(current_dog) { includes(:observed)
+                                    .where(observer_id: current_dog.id,
+                                    observed_id: current_dog.matches) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 7, 71 should be on its own line.